hw12-solid/src/main/java/ru/asamara56/atm/ATM.java

```diff
@@ -0,0 +1,28 @@
+package ru.asamara56.atm;
+
+import java.util.Map;
+
+public interface ATM {
+
+    /**
+     * Выдача купюр
+     * @param amount запрашиваемая сумма
+     * @return банкноты разного номиналаа
+     * @throws IllegalArgumentException если запрошенная сумма не может быть выдана
+     */
+    Map<Banknote, Integer> getMoney(int amount) throws IllegalArgumentException;
```

Comment:
`IllegalArgumentException` - непроверяемое исключение, его не надо здесь объявлять.

```suggestion
    Map<Banknote, Integer> getMoney(int amount);
```